src/AbstractServer.php

```diff
@@ -146,16 +134,13 @@ protected function _dispatch(Method\Definition $invokable, array $params)
                 $object = new $class();
             }
         }
+        // phpcs:disable
         return call_user_func_array([$object, $method], $params);
+        // phpcs:enable
     }
 
-    // @codingStandardsIgnoreStart
     /**
      * Map PHP type to protocol type
-     *
-     * @deprecated Since 2.7.0; method will be renamed to remove underscore
-     *     prefix in 3.0.
      */
-    abstract protected function _fixType(string $type): string;
-    // @codingStandardsIgnoreEnd
+    abstract protected function fixType(string $type): string;
```

Comment:
We might want to document the _purpose_ of this method, for those implementing it.

Reply:
Agreed. However, this is a documentation issue which is IMO out-of-scope of this PR. I opened an issue for that.